modules/mig/metadata.yaml

```diff
@@ -80,61 +80,83 @@ spec:
         location: examples/compute_instance/tags
   interfaces:
     variables:
-      - name: autoscaler_name
-        description: Autoscaler name. When variable is empty, name will be derived from var.hostname.
+      - name: project_id
+        description: The GCP project ID
```

Comment:
nit: The Google Cloud project ID

Reply:
Sure - These are auto-generated, so we'll need to update the source template.

Reply:
done - changed from chore to fix